chore: cow explorer improvements

mfw78 opened the pull request:
# Description

Extends the CoW Explorer section.

# Changes

- [x] View by order id (fill-or-kill and partially fillable)
- [x] View by address (show listing of orders)
- [x] View by batch (ie. transaction hash), showing orders and settlement graph

## Related Issues

Fixes #129

mfw78 committed:
chore: cow explorer improvements

alfetopito commented on docs/cow-protocol/tutorials/cow-explorer/order.md:
```diff
@@ -40,37 +41,30 @@ Below is a simple example of Order ID [`56ac15a9`](https://explorer.cow.fi/order
 
 ![Partially fillable order overview](/img/explorer/order_partial_overview.png)
 
+By clicking the 'View fills' button, the user can see the fills that have been executed for this order. Fills for the above example are show below:
```
Typo?

```suggestion
By clicking the 'View fills' button, the user can see the fills that have been executed for this order. Fills for the above example are shown below:
```

alfetopito commented on docs/cow-protocol/tutorials/cow-explorer/order.md:
```diff
@@ -40,37 +41,30 @@ Below is a simple example of Order ID [`56ac15a9`](https://explorer.cow.fi/order
-- **Limit price:** The price set when the order was placed (minus the slippage) & signed. This is the minimum price that you will receive.
-- **Execution price:** The actual price the order was executed at. This parameter can only be equal to or better than the limit price.
+- **Partially-filled:** State for orders that have been partially executed. Parts of the order have been executed and the corresponding funds transferred to the target account.
+- **Pre-signing:** State for orders that are to be pre-signed. Commonly used for orders submitted through the [CoW Swap](../cow-swap) `Safe` interface.
```
I would mention it's the case for any SC wallet

```suggestion
- **Pre-signing:** State for orders that are to be pre-signed. Commonly used for orders submitted from a Smart Contract wallet, such as the [Safe](https://safe.global/) wallet.
```

alfetopito commented on docs/cow-protocol/tutorials/cow-explorer/order.md:
```diff
@@ -40,37 +41,30 @@ Below is a simple example of Order ID [`56ac15a9`](https://explorer.cow.fi/order
-- **Open** (pending)**:** State for a standing order. Orders in this state are being considered by the solvers. It is the entry state for all orders, the default from the moment the order is placed. From here the order can transition to all other states
+As CoW Explorer is meant to help users visualize their signed orders, a critical parameter to show is the _state_ of the signed order. This parameter has the following status:
+
+- **Open** (pending)**:** State for a standing order. Orders in this state are being considered by the solvers. It is the entry state for all orders, the default from the moment the order is placed. From here the order can transition to all other states.
```
Would be worth to add a note about pre-signing orders?
For those types of orders, `Open` is not the entry state.

(Note, I don't know how to create foot notes, this is just a suggestion on the content)

```suggestion
- **Open** (pending)**:** State for a standing order. Orders in this state are being considered by the solvers. It is the entry state for all orders*, the default from the moment the order is placed. From here the order can transition to all other states.

^* Except for pre-sign orders, where the first state is [Pre-signing](link to pre-sign entry)
```

mfw78 replied:
Footnote added. Unfortunately can't link easily back up, but given that it's so close, should be OK.

alfetopito commented on docs/cow-protocol/tutorials/cow-explorer/address.md:
```diff
@@ -0,0 +1,31 @@
+
+![User address overview](/img/explorer/address_overview.png)
+
+By default, the explorer will show all orders that have been placed by the user, regardless of their status, in reverse chronological order. The user can filter the results by status, and sort the results by any of the columns.
```
> The user can filter the results by status, and sort the results by any of the columns.

Not really.
Not possible to filter nor sort in this view.

```suggestion
By default, the explorer will show all orders that have been placed by the user, regardless of their status, in reverse chronological order.
```

mfw78 replied:
Thanks for picking up the deliberate mistake :sweat_smile:

alfetopito commented on docs/cow-protocol/tutorials/cow-explorer/batch_auction.md:
```diff
@@ -0,0 +1,30 @@
+This is a visual representation of the routing that was used to settle the batch auction. The graph shows the orders that were settled in the batch auction, and the routing that was used to settle the batch auction.
+The graph is interactive, and the user can hover over any of the edges to see the amount that was settled between the respective nodes of the graph.
+
+![Batch Graph](/img/explorer/batch_graph.png)
```
I suggest mentioning the 2 types of visualization: transfer based and trade based, with a picture for each and a brief explanation of what each shows.

Also, why not using the `<Image>` component here like in other places?

mfw78 replied:
As the expert CoW Explorer aficionado, may I kindly request a PR on top of this one, covering the 2 types of visualizations?

alfetopito replied:
There you go https://github.com/cowprotocol/docs-v2/pull/251

alfetopito reviewed (commented):
A few suggestions

mfw78 committed:
Apply suggestions from code review

Co-authored-by: Leandro <alfetopito@users.noreply.github.com>

mfw78 committed:
chore: pre-sign footnote

alfetopito committed:
chore: cow explorer visualization (#251)

# Description

Follow up to
https://github.com/cowprotocol/docs-v2/pull/245#discussion_r1431308047

Add section about the visualization option on Explorer

# Changes

- [x] Add note about the different visualization options and how to
switch between them
- [x] Bonus: compress other explorer tutorial images

<!--
## Related Issues

Fixes #
-->

mfw78 committed:
fix: us english

mfw78 committed:
fix: minor typo and newlines
